src/KubeOps.Operator/Queue/EntityRequeueBackgroundService.cs

```diff
@@ -5,19 +5,19 @@
 using k8s;
 using k8s.Models;
 
-using KubeOps.Abstractions.Controller;
+using KubeOps.Abstractions.Reconciliation;
+using KubeOps.Abstractions.Reconciliation.Queue;
 using KubeOps.KubernetesClient;
 
-using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 
 namespace KubeOps.Operator.Queue;
 
-internal sealed class EntityRequeueBackgroundService<TEntity>(
+public class EntityRequeueBackgroundService<TEntity>(
```

Comment:
If made public (intentionally I guess?), then there should be an XML Documentation for the class to be consistent.